example/lib/star_wars/hooks.dart

```diff
@@ -0,0 +1,36 @@
+import 'package:flutter_hooks/flutter_hooks.dart';
+import 'package:flutter_hooks_gallery/star_wars/redux.dart';
+import 'package:flutter_hooks_gallery/star_wars/star_wars_api.dart';
+import 'package:provider/provider.dart';
+
+/// fetch planets and update store
+typedef FetchAndDispatchPlanets = Future<void> Function(String);
+
+/// return the redux store created by [useReducer}
+/// We use [Provider] to retrieve the redux store.
+Store<AppState, ReduxAction> useAppStore() {
+  final context = useContext();
+  return Provider.of<Store<AppState, ReduxAction>>(context);
+}
+
+/// return [AppState] hold by redux store.
```

Comment:
```suggestion
/// Returns [AppState] hold by redux's [Store].
///
```

Reply:
Actually, ignore this suggestion

The sentence in itself has the same issue has with `FetchAndDispatchPlanets`: not very useful